modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationTimeline.js

```diff
@@ -18,6 +19,7 @@ import {
 
 const PublicationTimeline = ({timelineItemsURL}) => {
 	const [timelineItems, setTimelineItems] = useState([]);
+	const [itemsFetched, setItemsFetched] = useState(false);
```

Comment:
Rename this to `loading` to be consistent with the naming we use in our change tracking components.